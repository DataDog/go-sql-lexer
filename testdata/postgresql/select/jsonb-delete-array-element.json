{
  "input": "SELECT data #- '{tags,0}' AS tags_without_first FROM products;",
  "outputs": [
    {
      "expected": "SELECT data #- ? FROM products",
      "statement_metadata": {
        "size": 14,
        "tables": [
          "products"
        ],
        "commands": [
          "SELECT"
        ],
        "comments": [],
        "procedures": []
      }
    },
    {
      "expected": "SELECT data #- '{tags,0}' FROM products",
      "normalizer_config": {
        "remove_space_between_parentheses": true
      },
      "obfuscator_config": {
        "keep_json_path": true
      }
    }
  ]
}