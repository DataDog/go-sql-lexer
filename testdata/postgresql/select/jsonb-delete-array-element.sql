SELECT data #- '{tags,0}' AS tags_without_first FROM products;
