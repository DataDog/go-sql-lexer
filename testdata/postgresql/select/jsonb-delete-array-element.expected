[
    {
        "expected": "SELECT data #- ? FROM products",
        "statement_metadata": {
            "size": 14,
            "tables": [
                "products"
            ],
            "commands": [
                "SELECT"
            ],
            "comments": [],
            "procedures": []
        }
    }
]